webplugin/gulpfile.js

```
@@ -370,24 +327,17 @@ const generateFilesByVersion = (location) => {
                 );
 
             for (var i = 0; i < pluginVersions.length; i++) {
-                var data = plugin.replace(
-                    ':MCK_PLUGIN_VERSION',
-                    pluginVersions[i]
-                );
+                var data = plugin.replace(':MCK_PLUGIN_VERSION', pluginVersions[i]);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'data'. Consider to delete it or rename it.

'data' is defined here:


(lint/suspicious/noRedeclare)